config.dev.yaml

```diff
@@ -28,11 +28,11 @@ metrics:
 services:
   - name: service-template-node
     # a relative path or the name of an npm package, if different from name
-    module: ./index.js
+    module: ./app.js
```

Comment:
bikesheding: I think "service.js" is more appropriate here, as it is not "app" that we are building but service.  server.js & service.js do sound similar though.